tests/unit/data.py

```diff
@@ -39,19 +43,25 @@
     'CHARACTER_VARYING': "lorem ipsum dolor",
     'TEXT': "sit amet, consectetur",
     'DATE': date(2001, 9, 11),
+    'TIMESTAMP': datetime(2019, 12, 12, 10, 11, 22, 333444),
```

Comment:
What will happen, if we put `tzinfo=timezone(offset=timedelta(hours=-2))` here

Reply:
nothing, postgreSQL keeps timestamp as int64 so we only need to convert `datetime` object into proper value. Python `datetime` class takes into account timezone shift during casting to timestamp so we are free to use the same function for both cases.